src/corelibs/U2Core/src/gobjects/AssemblyObject.h

```diff
@@ -34,6 +34,9 @@ class U2CORE_EXPORT AssemblyObject : public GObject {
 public:
     AssemblyObject(const QString& objectName, const U2EntityRef& ref, const QVariantMap& hints = QVariantMap());
 
+    /** Returns number of reads in the object. */
+    qint64 getReadCount(U2OpStatus& os) const;
```

Comment:
`getReadsCount`

Reply:
Nope, getReadCount is the correct variant.